src/components/basic_table/basic_table.tsx

```diff
@@ -1382,26 +1378,22 @@ export class EuiBasicTable<T = any> extends Component<
         not configured. This callback must be implemented to handle pagination changes`);
       }
 
-      let ariaLabel: ReactElement | undefined = undefined;
-
-      if (tableCaption) {
-        ariaLabel = (
-          <EuiI18n
-            token="euiBasicTable.tablePagination"
-            default="Pagination for preceding table: {tableCaption}"
-            values={{ tableCaption }}
-          />
-        );
-      }
-
       return (
-        <PaginationBar
-          aria-controls={this.tableId}
-          pagination={pagination}
-          onPageSizeChange={this.onPageSizeChange.bind(this)}
-          onPageChange={this.onPageChange.bind(this)}
-          aria-label={ariaLabel}
-        />
+        <EuiI18n
+          token="euiBasicTable.tablePagination"
+          default="Pagination for preceding table: {tableCaption}"
```

Comment:
What do you think about removing the word "preceding" ? I'm envisioning a case where screen readers are announcing the landmarks menu, or users are navigating by the landmarks shortcut key. Users may not have a sense of space or visual relationship, so saying `Pagination for table: {tableCaption}` might be more descriptive.

Reply:
I'm good with it! Although as a disclaimer I didn't write the original copy so I have no strong stakes either way haha, but the landmark use case you mention makes a lot of sense.
```suggestion
          default="Pagination for table: {tableCaption}"
```